### PR TITLE
Add recommendation for X509Certificate2.PrivateKey

### PR DESCRIPTION
**Pull request by mjrousos**
@bartonjs

### CODE REVIEW

File: docs/RecommendedChanges/System.Security.Cryptography.X509Certificates/Use GetRSAPrivateKey or GetECDsaPrivateKey.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+### Recommended Action
+Use GetRSAPrivateKey() or GetECDsaPrivateKey() extension methods instead.
```
Comment (twsouthwick) [inline docs/RecommendedChanges/System.Security.Cryptography.X509Certificates/Use GetRSAPrivateKey or GetECDsaPrivateKey.md]: Is this in the same assembly or does one need to be added?

Comment (mjrousos) [inline docs/RecommendedChanges/System.Security.Cryptography.X509Certificates/Use GetRSAPrivateKey or GetECDsaPrivateKey.md]: In .NET Core, the extension is in the same contract so it should be easy to find. Curiously, on desktop .NET the extension is in System.Core whereas the rest of X509Certificate2 is in System.

Comment (twsouthwick) [inline docs/RecommendedChanges/System.Security.Cryptography.X509Certificates/Use GetRSAPrivateKey or GetECDsaPrivateKey.md]: Ok - that's probably fine. We had a problem with some recommended changes that recommended an extension method that required adding a separate package and a user couldn't find it.

**Comment by twsouthwick** [comment]
LGTM

**Comment by bartonjs** [comment]
LGTM

**Pull request merged**